universal-application-tool-0.0.1/app/services/cloud/aws/SimpleStorage.java

```diff
@@ -66,13 +66,14 @@ public SimpleStorage(
         });
   }
 
-  public URL getPresignedUrl(String key) {
+  @Override
+  public String getPresignedUrlString(String key) {
```

Comment:
Doc this similar to the above.  EG what's a Key here compared to a fileKey (which is doc'd) above.